packages/application-generic/src/factories/push/handlers/ntfy.handler.ts

```diff
@@ -0,0 +1,20 @@
+import { ChannelTypeEnum, ICredentials } from '@novu/shared';
+import { NtfyPushProvider } from '@novu/ntfy';
+import { BasePushHandler } from './base.handler';
+
+export class NtfyHandler extends BasePushHandler {
+  constructor() {
+    super('ntfy', ChannelTypeEnum.PUSH);
+  }
+
+  buildProvider(credentials: ICredentials) {
+    if (!credentials.baseUrl || !credentials.topic) {
+      throw Error('Config is not valid for ntfy');
+    }
```

Comment:
```suggestion
    if (!credentials.topic) {
      throw Error('Topic is required for ntfy');
    }
```

<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** As `baseUrl` is optional, it shouldn't throw error when `baseUrl` is empty‏